docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -9,13 +9,13 @@ aliases:
   - ../../../enterprise/saml/set-up-saml-with-okta/
   - ../../../enterprise/saml/troubleshoot-saml/
 description: Learn how to configure SAML authentication in Grafana's configuration file.
-menuTitle: Configure SAML using the configuration file
+menuTitle: SAML
```

Comment:
So one is called SAML user interface. Should we call this one SAML config file? Or just leave it?

Reply:
I think maybe just leave it, otherwise it sounds a bit like we're talking about a SAML specific config file (instead of the general Grafana config file).